docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM quay.io/pypa/manylinux1_x86_64
```

Comment:
would be nice to see from the filename or path that this is a dockerfile for this case. So I suggest to either create another subdirectory 'manylinux' or call it 'Dockerfile_manylinux' (not sure what the best practices are for docker)